src/web/statics.rs

```diff
@@ -16,10 +15,11 @@ const RUSTDOC_CSS: &str = include_str!(concat!(env!("OUT_DIR"), "/rustdoc.css"))
 const STATIC_SEARCH_PATHS: &[&str] = &["vendor/pure-css/css", "static"];
 
 pub(crate) fn static_handler(req: &mut Request) -> IronResult<Response> {
-    let router = extension!(req, Router);
-    let file = cexpect!(req, router.find("file"));
+    let mut file = req.url.path();
+    file.drain(..2).for_each(std::mem::drop);
+    let file = file.join("/");
```

Comment:
What does the `/` do?

Reply:
`req.url.path()` is a `Vec<&str> of path segments, so the join turns it back into a path.